grub.d/default/extra/mender-offline-grub-probe-helper.cfg

```diff
@@ -0,0 +1,33 @@
+# This script is a helper to make it possible to run `grub-install` and
+# `update-grub` in a container or chroot. This is necessary when making a build
+# time image using mender-convert. GRUB provides a way to swap out its
+# `grub-probe` tool, which is what this script uses to replace live probes with
+# preconfigured values.
+
+# The offline mode is designed to run in a container or in chroot, never outside.
+if type systemd-detect-virt 2>&1 >/dev/null && \
+    ! systemd-detect-virt --container >/dev/null && \
+    ! systemd-detect-virt --chroot >/dev/null; then
+
+    echo "The /etc/default/grub.d/mender-offline-grub-probe-helper.cfg file is only meant to run in a container or chroot as part of a build process. Please remove it before running on a real device." 1>&2
```

Comment:
No, because the installed path is not the same as the source path. It's in the `extra` source folder because it is normally only installed when using mender-convert, and even then, only temporarily. So if it is still here when running on a real device, someone made a mistake, and it should be deleted from the mentioned path.